### DISCUSSION TITLE
Render the residential areas

### DISCUSSION THREAD

**Discussion by severinmenard** [organicmaps/organicmaps#7584] [Ideas]
Why are the closed ways or relations tagged with landuse=residential not rendered? In many contexts where it is the only landcover category currently mapped, the Organic Maps rendering shows only streets, without providing further clues about the extent of residential areas.

**Comment by Misalf-git** [comment]
Maybe because the map looks cleaner without.

There is no consensus of whether the boundaries of residential areas should be mapped at adjacent road centers, road edges, property edges or something else, resulting in creative sidewalk mapping at one place and profane rectangle-thrown-at-a-district somewhere else.

**Comment by severinmenard** [comment]
Or emptier and less informative... :) IMHO the way it is done in OSM carto (the default OSM rendering) is pretty neat.
Regarding the lack of consensus on where the residential areas should start or stop, this is basically the same for any land cover area. And yet they are rendered in Organic Maps.

**Comment by Misalf-git** [comment]
Yeah, I don't know. Would be interesting to actually see in OM and judge from there.

I guess the qeuestion should be how valuable / informative it is to render residential areas, compared to visual load / confusion it may create.

**Comment by pastk** [comment]
- #2184
